examples/dt/neutron/3cont_2comp-ipv6-all-geneve-dvr-stf-rally

```diff
@@ -0,0 +1,62 @@
+# Deployed Topology 3cont_2comp-ipv6-all-geneve-dvr-stf-rally
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+|           | ovn-router-flavors           |                    |
+| Glance    | default                      | Must have          |
+| Cinder    | default                      | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+
+### Additional information 
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+add configuration of ipv6, OVNEncapTos=inherit
+
+## Considerations/Constraints
+
+
+## Testing tree
+Tempest and Rally
+
+| Test framework   | Stage to run | Special configuration | Test case to report |
+| ---------------- | ------------ | --------------------- | :-----------------: |
+| Rally            | stage8       |                       |      11111111       | 
+| Tempest/neutron  | stage7       | Use rhel image        |      11223344       |
+| Tempest/scenairo | stage7       | Use cirros image      |      22334455       |
+
+## Stages
+
+All stages must be executed in the order listed below. Everything is required unless otherwise indicated.
+
+1. [Install dependencies for the OpenStack K8S operators](stage1)
+2. [Install the OpenStack K8S operators](stage2)
+3. [Configuring networking on the OCP nodes](stage3)
+4. [Configure and deploy the control plane](stage4)
+5. [Configure and deploy the initial data plane to prepare for CephHCI installation](stage5)
```

Comment:
is ceph really required? or just there for some reusability point? is ceph used to be there in mentioned 17.1 jobs?

Reply:
Marking this as Resolved, because this is mapped to unified job uni-delta-ipv6